pkg/test/runner/runner.go

```diff
@@ -19,6 +19,11 @@ type Runner struct {
 }
 
 const (
+	// If this env is set to "true", this e2e test framework will update the
+	// expected diff and results if they already exist. If will not change
+	// config.yaml.
+	updateExpectedEnv string = "KPT_E2E_UPDATE_EXPECTED"
```

Comment:
I will be nice to have a full command somewhere (in README or a comment in Makefile) that a developer can copy and run to update the expected.

Reply:
+1